server/helpers/verifyUser.js

```diff
@@ -25,5 +27,53 @@ class VerifyUser {
     }
     return done('password or email is incorrect');
   }
+
+
+  /**
+     * Retrieve hashed the password.
+     * @param {string} aT The user's token offered by google to be checked.
+     * @param {string} rT The user's token offered by google to be checked.
+     * @param {string} profile The user info
+     * @param {string} done function supplying user credentials after user is successfully checked.
+     * @returns {boolean} message or User details after passing all checks.
+     */
+  static async inSignSocial(aT, rT, profile, done) {
```

Comment:
Update the method name to **socialSignIn**

Reply:
Feedback Implemented!